docs/src/DeveloperDocumentation/debugging.md

```diff
@@ -24,12 +24,14 @@ object in the system.
 The simplest example is the creation of a polynomial ring. If we mutate the
 array of symbols used for printing, we have effectively changed the ring.
 
-```
+```julia-repl
```

Comment:
Had to change this back from `jldoctest` because the example actually does not work.

Do we have any other similar example? If yes, we can use that. If not, perhaps this section should be removed...

CC @lgoettgens @thofma @fieker

Reply:
Actually my analysis was wrong, this still can be reproduced but I can't do it in CI because the cached ring is modified and then running other tests break

Reply:
Could you maybe pass `cached=false` to not cache the broken ring?

Reply:
Yes, indeed. I misunderstood the problem last night.